src/attributes.js

```diff
@@ -12,9 +12,31 @@ export class AttributesCustomAttribute {
   }
 
   valueChanged() {
-    Object.keys(this.value).forEach(attribute => {
+    Object.keys(normalizeAtttibutes(this.value)).forEach(attribute => {
       this.element.setAttribute(attribute, this.value[attribute]);
     });
   }
 
 }
+
+/**
+ * @param {object|string|string[]} value
+ * @returns {object} where all the values are strings or boolean
+ */
+function normalizeAtttibutes(value, result = {}) {
```

Comment:
that result = {} is for some future usage? normalizeAtttibutes isn't exported atm, so if not, plain old let result ={} would be preferable